documentation/commands/git-town.md

```diff
@@ -12,6 +12,7 @@ git town hack-push-flag [(true | false)]
 git town help
 git town install-fish-autocompletion
 git town main-branch [<branch_name>]
+git town parent-branch <child_branch_name> <parent_branch_name>
```

Comment:
Well, only feature branches have parents in the Git Town universe, so I could consider this implicit. Speaking of which, we should guard against setting the parent branch of a perennial branch.